proto/einride/iam/v1/annotations.proto

```diff
@@ -2,14 +2,6 @@ syntax = "proto3";
 
 package einride.iam.v1;
 
-option csharp_namespace = "Iam.V1";
```

Comment:
Are these all removed by the linter? Same goes for the other proto files

Reply:
They are no longer needed when using managed mode. With managed mode, the file options are configured as part of the code generation config and not the schema definition.

See here:

https://docs.buf.build/tour/use-managed-mode/

Reply:
Sweet!